phabfive/maniphest.py

```diff
@@ -2133,26 +122,41 @@ def info(self, task_id):
 
     def create_from_config(self, config_file, dry_run=False):
         if not config_file:
-            raise PhabfiveException("Must specify a config file path")
+            raise PhabfiveException(f"Must specify a config file path")
 
-        if not Path(config_file).is_file():
+        if Path(config_file).is_file():
             log.error(f"Config file '{config_file}' do not exists")
-            return
 
         with open(config_file) as stream:
-            yaml_loader = YAML()
-            root_data = yaml_loader.load(stream)
+            root_data = yaml.load(stream, Loader=yaml.Loader) # nosec-B506
 
         # Fetch all users in phabricator, used by subscribers mapping later
-        users_query = self.phab.user.search()
+        # Extended support for phabricator instances with more than 100 (i.e. deftault query limit) users
+        # BUT with limieted exception handling e.g. retries needed when phabricator instance is not responding to API calls
```

Comment:
These comments could be removed. They only serve a purpose for the reviewer of this PR